papis/commands/add.py

```diff
@@ -421,7 +419,7 @@ def run(paths: List[str],
         for d_path in tmp_document.get_files():
             papis.utils.open_file(d_path)
     if confirm:
-        if not papis.tui.utils.confirm("Really add?"):
+        if not papis.tui.utils.confirm("Do you want to add the new document anyway?"):
```

Comment:
```suggestion
        if not papis.tui.utils.confirm("Do you want to add the new document?"):
```
? Since this should show up during normal working conditions (not just if there's a duplicate or something)

Reply:
Oh true, I hadn't considered that. Thanks!